lstchain/tools/pedestal_param.json

```diff
@@ -8,19 +8,19 @@
      "log_level":"DEBUG"
    },         
    "EventSource": {
-      "input_url": "/ctadata/franca/LST/LST-1.1.Run00472.0000.fits.fz",
+      "input_url": "/ctadata/franca/fefs/aswg/real/R0/20191107/LST-1.1.Run01579.0000.fits.fz",
```

Comment:
change url please

Reply:
This is an example, people must change it with their path... what you want I put?

Reply:
There is no default in La Palma ?
If not it's fine.